SonarQube.Client.Tests/Requests/Api/V7_20/GetIssuesRequestWrapperTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SonarQube.Client.Api.V7_20;
+using SonarQube.Client.Tests.Infra;
+using static SonarQube.Client.Tests.Infra.MocksHelper;
+
+namespace SonarQube.Client.Tests.Requests.Api.V7_20
+{
+    [TestClass]
+    public class GetIssuesRequestWrapperTests
```

Comment:
`GetIssuesRequestWrapper` is a helper class that internally makes three requests to `GetIssues`, first for code smells, then for bugs and finally for vulnerabilities.
This was a hacky way to partially work round a limitation in the SQ/SC API that meant a request could only return a maximum of 10000 results.

Previously, there weren't any direct tests for this class, although it was tested indirectly by the "GetSuppressedIssuesAsync" tests.

I've added one simple test to make sure that three requests are made and the expected query parameters are set.